code/game/machinery/fax.dm

```diff
@@ -159,9 +160,15 @@
 		to_chat(user, "<span class='notice'>You [anchored ? "wrench" : "unwrench"] \the [src].</span>")
 
 
+/obj/machinery/faxmachine/proc/end_cooldown()
+	sendcooldown = FALSE
+
+
+/obj/machinery/faxmachine/cic
+	department = "Combat Information Center"
```

Comment:
This won't work without changing the rest of the fax code.

Reply:
Could you explain?

Reply:
Check the topic.dm on faxreply and faxcreate and check the other faxes file in the admin folder.